cli/src/commands.tsx

```diff
@@ -202,14 +200,41 @@ const UserInput = () => {
   )
 }
 
-export const CommandProvider = ({ children }: PropsWithChildren) => {
-  const [input, setInput] = useState<string>(CommandKeys.Help)
+export interface CommandProviderProps extends MainframeProps {
+  children: React.ReactNode
+}
+
+export const OptionContext = createContext<Partial<Record<OptionKey, string>>>({
```

Comment:
Related to the other comments - options should probably be processed via yarg, outside React's context